Make compat tables more clear (#27)
10gen/docs-shared: dbx/driver-test-note.rst

.. important::

>    The check marks (✓) indicate the driver supports all features for the listed MongoDB version.

Reviewer: Issue: the descriptive text describes representing a specific single version, so the subject should match.

```suggestion
   A check mark (✓) indicates the driver supports all features for the listed MongoDB version.
```

>    Any driver version that is marked as compatible with MongoDB version 4.0 and later are
>    tested to ensure they continue to work with newer versions of MongoDB. However, the

Reviewer: Issue: "Any" refers to a singular noun, so the rest of the sentence that refers to the subject should continue to refer to a singular item.

```suggestion
   Any driver version that is marked as compatible with MongoDB version 4.0 and later is
   tested to ensure it continues to work with newer versions of MongoDB. However, the
```
